pkg/cmd/repo/setdefault/setdefault_test.go

```diff
@@ -175,17 +176,21 @@ func TestDefaultRun(t *testing.T) {
 					Repo:   repo1,
 				},
 			},
-			wantStdout: "no default repository has been set; use `gh repo set-default` to select one\n",
+			wantStdout: "",
+			wantStderr: "no default repository has been set; use `gh repo set-default` to select one\n",
 		},
 		{
 			name: "view mode no current default",
+			tty:  false,
 			opts: SetDefaultOptions{ViewMode: true},
 			remotes: []*context.Remote{
 				{
 					Remote: &git.Remote{Name: "origin"},
 					Repo:   repo1,
 				},
 			},
+			wantStdout: "",
```

Comment:
We can remove this empty string

Reply:
Thank you for the additional feedback. I have updated the code based on your suggestions.